### PR TITLE
Basic scripts

### PR DESCRIPTION
**Pull request by bretttegartms**

### CODE REVIEW

**Comment by bretttegartms** [comment]
@ceache @andreikeis Ready for review

File: treadmill-scripts/newserver.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+cell=$1
```
[the remark this answers is not in the document]
Comment (bretttegartms) [reply treadmill-scripts/newserver.sh]: OK, updated script.

File: treadmill-scripts/newcell.sh [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+# Define docker_registry for cell.
+cat << E%O%F > $tmpdir/$TREADMILL_CELL.json
+{docker_registry: '$DOCKER_REGISTRY'}
+E%O%F
+
+# Define cell in LDAP
+treadmill admin ldap cell configure $TREADMILL_CELL \
+	--version 3.7 \
+	--root /var/lib/treadmill-master \
```
[the remark this answers is not in the document]
Comment (bretttegartms) [reply treadmill-scripts/newcell.sh]: OK, removed variable.

File: treadmill-scripts/newcell.sh [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+done
+echo -e "All servers have started Zookeeper.\n"
+
+# Check Zookeeper quorum
+for i in `seq 1 3`
+do
+	# Check ZK leader/follower status
+	until echo mntr | nc -w 5 $TREADMILL_CELL-zk-$i 2181 2>/dev/null | grep -q 'zk_server_state'
+	do
+		echo Waiting for $TREADMILL_CELL-zk-$i to signal quorum joined...
```
[the remark this answers is not in the document]
Comment (bretttegartms) [reply treadmill-scripts/newcell.sh]: OK, changed formatting.

File: treadmill-scripts/newcell.sh [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+until treadmill sproc --cell $TREADMILL_CELL scheduler --once > /dev/null 
+do
+	sleep 5
+	echo -e 'Bootstrapping namespace...\n'
+done
+echo -e 'Zookeeper namespace initialized.\n'
+
+# Start ticket locker
+echo -e 'Please log in to IPA server and run the following commands to start cell ticketlocker:\n'
+echo "cat << E%O%F > /etc/ticketlocker/$TREADMILL_CELL
```
[the remark this answers is not in the document]
Comment (bretttegartms) [reply treadmill-scripts/newcell.sh]: OK, changed input.

File: treadmill-scripts/newcell.sh [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+	cat << E%O%F > $tmpdir/$TREADMILL_CELL.yaml
+---
+treadmill_cell: $TREADMILL_CELL
+treadmill_ldap: $TREADMILL_LDAP
+treadmill_ldap_suffix: $TREADMILL_LDAP_SUFFIX
+treadmill_dns_domain: $TREADMILL_DNS_DOMAIN
+treadmill_isa: node 
+E%O%F
+	subnetn='subnet'$i
+	hostname=`treadmill admin aws instance create \
```
[the remark this answers is not in the document]
Comment (bretttegartms) [reply treadmill-scripts/newcell.sh]: Changed formatting.

File: treadmill-scripts/delcell.sh [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cellname=$1
+
+treadmill admin ldap cell delete $cellname
+./delserver.sh $cellname-zk-{1..3}
```
Comment (andreikeis) [inline treadmill-scripts/delcell.sh]: dont use relative path. one simple option is:

${0%/*}/delserver.sh

File: treadmill-scripts/newcell.sh [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+cat << E%O%F > $tmpdir/$TREADMILL_CELL.json
+{docker_registry: '$DOCKER_REGISTRY'}
+E%O%F
+
+# Define cell in LDAP
+treadmill admin ldap cell configure $TREADMILL_CELL \
+	--version 3.7 \
+	--root /var/lib/treadmill-master \
+	--username $PROID \
+	--location local.local \
```
[the remark this answers is not in the document]
Comment (bretttegartms) [reply treadmill-scripts/newcell.sh]: This is written in Bash- I don't want to add a dependency on awscli to run this script. Added --location to CLI options to allow user to describe location, defaulting to local.local.

File: treadmill-scripts/newcell.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (andreikeis) [reply treadmill-scripts/newcell.sh]: i do not see call to treadmill admin cell --cell $CELL configure-dns...

File: treadmill-scripts/newserver.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreikeis) [reply treadmill-scripts/newserver.sh]: i do not see the change, what am i missing? can be done later.

**Pull request merged**